core/mondoo-azure-security.mql.yaml

```diff
@@ -419,7 +419,7 @@ queries:
             && properties.sourceAddressPrefix == /\*|0\.0\.0\.0|<nw>\/0|\/0|internet|any/
         )
       allNsgTCP.all(properties.destinationPortRange != "*")
-      props.disallowedPortsRDP {
+      props.disallowedPortsRDPMondoo {
```

Comment:
Thanks Manuel. I couldn't get why you changed the Azure properties name as we already considered the uniqueness of properties's name in the first place. 

Beside if you justify the reasoning, then the name should also be reflected in the query!!! 

example:
```
allNsgTCP.none(destinationPortRange.any(fromPort <= disallowedPortRDP && toPort >= disallowedPortRDPMondoo))
```